apps/wallet/src/ui/app/staking/stake/StakeForm.tsx

```diff
@@ -56,9 +72,21 @@ function StakeForm({
         setFieldValue('amount', maxToken);
     }, [maxToken, setFieldValue]);
 
+    const calculateRemaining = useMemo(() => {
+        if (!coinBalance) return 0n;
+        const bigIntAmount = parseAmount(amount, coinDecimals);
+        return coinBalance - bigIntAmount - gasBudget;
```

Comment:
Just use 
```suggestion
        return totalAvailableBalance - bigIntAmount;
```

Reply:
Also about this do we really want to change the available balance based on the amount that the user entered to stake? In other places eg send tokens we show always the total balance of the user.
cc @mystie711